src/Aks/readme.md

```diff
@@ -70,9 +70,9 @@ module-name: $(prefix).$(service-name)
 namespace: Microsoft.Azure.PowerShell.Cmdlets.$(service-name)
 clear-output-folder: true
 output-folder: .
-aks: $(repo)/specification/containerservice/resource-manager/Microsoft.ContainerService
```

Comment:
What's the reason of updating Aks? We are not using generated version of Aks but using hand writing one.

Reply:
Extract from an Email thread
---------------------------------------------------------------------------
From: Xiaogang Ding 
Sent: Wednesday, October 21, 2020 7:52 PM
To: Shaila Hirji ; Yunchi Wang; Yeming Liu ; Dingmeng Xue 
Cc: Karthik Kattamudi ; Nataraj Srikantaiah 
Subject: RE: Pipeline Check failure for Service:ResourceMover 

Hi Shaila,
 
Thanks for pointing that out. Resource Mover team moved the swagger location but did not change it in readme.md. I have changed it and retrigger the CI. 
 
Thanks,
Xiaogang
 
From: Shaila Hirji 
Sent: Thursday, October 22, 2020 5:40 AM
To: Xiaogang Ding ; Yunchi Wang; Yeming Liu; Dingmeng Xue 
Subject: Pipeline Check failure for Service:ResourceMover
 
Hi, 
 
I have been encountering this error : Error: Could not read 'https://github.com/Azure/azure-rest-api-specs/tree/master/specification/regionmove/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/regionmovecollection.json'. and not sure which file needs to be modified and to what. The failure seems to be while accessing some file in the azure-rest-api repo but the version may have been outdated. 
 
Here is the PR: https://github.com/Azure/azure-powershell/pull/13194
 
I did notice src/Aks/readme.md was pointing to something that was no longer part of the rest-api repo:  
 
So I changed it to this:  

But don’t think this resolved the error. Let me know if I should revert it back to the original one. 
 
Please assist with this as it is blocking us from making this available in selfhost. 
 
Thanks, 
 
Shaila

Reply:
I think we need to revert it back to the original one.